Kernel/Heap/Heap.h

```diff
@@ -68,40 +68,56 @@ class Heap {
         return needed_chunks * CHUNK_SIZE + (needed_chunks + 7) / 8;
     }
 
-    void* allocate(size_t size, CallerWillInitializeMemory caller_will_initialize_memory)
+    void* allocate(size_t size, size_t alignment, CallerWillInitializeMemory caller_will_initialize_memory)
     {
+        // We can't align anything less than CHUNK_SIZE, presumably due to edge cases in the bitmap calculation.
+        // Not that it really matters, we track only chunks, nothing smaller.
```

Comment:
The "presumably..." part of this comment adds confusion and makes it seem less trustworthy. Something like "The minimum possible alignment is CHUNK_SIZE" would suffice IMO. :)